ethereum_history_api/circuits/lib/src/serde.nr

```diff
@@ -111,3 +102,75 @@ impl Serde<STORAGE_BLOCK_LEN> for StorageWithinBlock<1> {
         }
     }
 }
+
+global RECEIPT_MAX_DEPTH_NO_LEAF = 6;
+global RECEIPT_NODES_LEN = 3192; // MAX_NODE_LEN * MAX_DEPTH_NO_LEAF;
+
+pub fn serialize_nodes(nodes: [Node; RECEIPT_MAX_DEPTH_NO_LEAF]) -> [Field; RECEIPT_NODES_LEN] {
```

Comment:
Why isn't it Serde implementation?

Reply:
Oh, I get it. But maybe it should be a structure?

Reply:
I don't think so, because we want to use it to write Serde for Proof. If we make it a structure it would overcomplicate Proof's definition